examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -63,7 +61,7 @@ class FeederActor extends Actor {
   def receive: Receive = {
     case SubscribeReceiver(receiverActor: ActorRef) =>
       println("received subscribe from %s".format(receiverActor.toString))
-      receivers = LinkedList(receiverActor) ++ receivers
+      receivers = MutableList(receiverActor) ++ receivers
```

Comment:
If you retain a Scala collection here, it's more direct / efficient in theory to write `receiverActor +=: receivers` to prepend. In practice, I don't know what people find easier to understand.